interface/ceed-jit-tools.c

```diff
@@ -160,8 +165,14 @@ int CeedLoadSourceToInitializedBuffer(Ceed ceed, const char *source_file_path, c
           CeedCall(CeedFree(&ceed_relative_path));
         }
         // ---- Recursive call to load source to buffer
-        CeedDebug256(ceed, CEED_DEBUG_COLOR_SUCCESS, "JiT Including: %s\n", include_source_path);
-        CeedCall(CeedLoadSourceToInitializedBuffer(ceed, include_source_path, buffer));
+        for (CeedInt i = 0; i < *num_files; i++) is_included |= !strcmp(include_source_path, (*filepaths)[i]);
```

Comment:
TIL that `|=` is a thing.

Reply:
Just note that it's bitwise or (not logical or), and that C implicit conversion rules are permissive and sometimes surprising.
```
bool b = false;
b |= 2; // b now has value 1
b = 2; // b now has value 1
b &= 2; // b now has value 0
```